src/pages/api/selectJourneyDirection.ts

```diff
@@ -0,0 +1,38 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { getDomain, setCookieOnResponseObject, redirectTo, redirectToError } from './apiUtils/index';
+import { isSessionValid } from './service/validator';
+import { JOURNEY_COOKIE } from '../../constants';
+import { inboundErrorId, outboundErrorId } from '../selectJourneyDirection';
+
+export default (req: NextApiRequest, res: NextApiResponse): void => {
+    try {
+        if (!isSessionValid(req, res)) {
+            throw new Error('Session is invalid.');
+        }
+
+        const { inboundJourney, outboundJourney } = req.body;
+
+        if (inboundJourney && outboundJourney) {
```

Comment:
Do we want to do anything to check if what they have said the inboundJourney and outboundJourney is valid? Currently this API will accept any values, unsure if we want to throw an error if its not one of the valid choices for what they could select. We could reuse the `const rawService: RawService = await getServiceByNocCodeAndLineName(operatorInfo.nocCode, lineName);
    const service: Service = {
        ...rawService,
        journeyPatterns: await enrichJourneyPatternsWithNaptanInfo(rawService.journeyPatterns),
    };` methods from the props?

Reply:
as per conversation we wont do this for now as not beneficial due to the cookies which could get changed